pages/installing-packages.md

```diff
@@ -0,0 +1,110 @@
+
+The software in conda-forge are not subject to Anaconda's licensing restrictions, and the conda package manager is free software, so both can be used without any legal restrictions in national labs and universities.
+
+Until recently, the (relatively) [hard part](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) had been to ensure that you're using conda-forge, rather than an Anaconda default channel. The instructions below describe how to install [Miniforge](https://github.com/conda-forge/miniforge), which is conda-forge without the Anaconda default channel.
+
+You likely have a package manager for your operating system, such as Homebrew, apt-get, or yum. Use conda for your Python packages and your operating system's package manager for applications (web browsers, text editors, etc.).\*
+
+(\* I'm doing conda a disservice by describing conda as a Python package manager, though [it does much more](https://jakevdp.github.io/blog/2016/08/25/conda-myths-and-misconceptions/#Myth-#2:-Conda-is-a-Python-package-manager), for the sake of keeping this simple.)
+
+# What is "mamba"?
+
+We recommend using "mamba", which is a drop-in replacement for "conda" that is many times faster. You particularly notice it when a package has many dependencies or complex version constraints on its dependencies.
+
+In fact, the conda developers are [incorporating mamba into conda](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community), though at the time of this writing, that integration is still experimental. These instructions will describe how to use mamba directly.
+
+# Where will the files go?
+
+The entire Python distribution, with all packages and the shared libraries that support them, will go in a new directory, most likely in your home directory and named `mambaforge`. All of the files in it are installed with your own user permissions (i.e. not superuser/`sudo`).
+
+## How to get rid of them if you change your mind
+
+1. Delete that directory with `rm -rf ~/mambaforge`.
+2. Delete a file named `~/.condarc`, if you have one.
+3. Check your shell configuration file, probably named `~/.bashrc`, for a "`>>> conda initialize`" section. If you have one, delete it.
+
+Those three steps will remove any vestige of the conda installation.
+
+## How to save an old package list before deleting it
```

Comment:
This would be better as a details dropdown. I can change it if you'd like.

Reply:
Please do! (I don't know the syntax.)